app/helper.py

```diff
@@ -0,0 +1,17 @@
+# Validate if GET by Planet ID does not exist
+from flask import Blueprint, jsonify, request, make_response, abort
+from app.models.planet import Planet
+
+
+def validate_planet(planet_id):
```

Comment:
Nice work pulling this into a helper to keep your routes short and sweet. In the future, you might need to make 'helper.py' more. Think about Task List, you have Goal and Task, you may need helpers for each. If you refactor helper methods into a helper file then you could have goal_helper.py and task_helper.py